### ISSUE TITLE
KeyError on _panel_positions[panel] in editorlayout.py line 213

### ISSUE THREAD

**Issue by Torxed** [jliljebl/flowblade#1060]
Please include the following information.
1) Flowblade version (Help->About):
```
numpy version: 1.22.3
Application version: 2.8.0
OS: Arch Linux
Python 3.10.4 (main, Mar 23 2022, 23:05:40) [GCC 11.2.0]
GTK+ version: 3.24.34
XDG user data exists.
prefs updated to new version, new param count: 95
Locale: en_US
Translations at /usr/share/locale were not found, using program root directory translations.
Use OS locale language.
Valid shortcut files found: ['flowblade.xml', 'premiere.xml']
Keyboard shortcuts file: flowblade.xml
Gtk version is 3.24.34, Flowblade theme is available.
Using Full Screen size for layout: 5120 x 1440
Small height: False
Small width: False
mlt_repository_init: failed to dlopen /usr/lib/mlt/libmltopengl.so
  (libmovit.so.8: cannot open shared object file: No such file or directory)
mlt_repository_init: failed to dlopen /usr/lib/mlt/libmltrtaudio.so
  (librtaudio.so.6: cannot open shared object file: No such file or directory)
mlt_repository_init: failed to dlopen /usr/lib/mlt/libmltsox.so
  (libsox.so.3: cannot open shared object file: No such file or directory)
Detecting environment...
```
2) MLT version (Help->Runtime Environment):
```
extra/mlt 7.6.0-1 [installed]
    An open source multimedia framework
```
3) Your distribution (Ubuntu, Debian, Mint etc.):
Arch Linux *(Linux bigrigv2 5.17.9-arch1-1 # 1 SMP PREEMPT Wed, 18 May 2022 17:30:11 +0000 x86_64 GNU/Linux)*
4) If possible, try to provide step-by-step instructions on how to reproduce the issue. From developer point of view this is the easieast way to understand what is happening.:

```
$ sudo pacman -S flowblade mlt
$ flowblade
```
And it generates:
```
Traceback (most recent call last):
  File "/usr/bin/flowblade", line 93, in <module>
    app.main(modules_path)
  File "/usr/lib/python3.10/site-packages/Flowblade/app.py", line 307, in main
    create_gui()
  File "/usr/lib/python3.10/site-packages/Flowblade/app.py", line 487, in create_gui
    editor_window = editorwindow.EditorWindow()
  File "/usr/lib/python3.10/site-packages/Flowblade/editorwindow.py", line 166, in __init__
    self._init_view_menu(self.ui.get_widget('/MenuBar/ViewMenu'))
  File "/usr/lib/python3.10/site-packages/Flowblade/editorwindow.py", line 1081, in _init_view_menu
    panel_positions_menu_item = editorlayout.get_panel_positions_menu_item()
  File "/usr/lib/python3.10/site-packages/Flowblade/editorlayout.py", line 398, in get_panel_positions_menu_item
    compositors_panel_menu =  _get_position_selection_menu(appconsts.PANEL_COMPOSITORS)
  File "/usr/lib/python3.10/site-packages/Flowblade/editorlayout.py", line 425, in _get_position_selection_menu
    current_position = _get_panel_position(panel_id)
  File "/usr/lib/python3.10/site-packages/Flowblade/editorlayout.py", line 213, in _get_panel_position
    return _panel_positions[panel]
KeyError: 2
```

**Comment by vincens2005** [comment]
I'm having the same issue, similar system

**Comment by aaruni96** [comment]
I have the same issue

**Comment by jliljebl** [comment]
Fixed in repo will be working in upcoming 2.10.
